Makefile

```diff
@@ -1,5 +1,18 @@
 .DEFAULT_GOAL := help
 
+CONDITION_ORC_PORT=9001
+ALLOY_PORT=9091
+HSS_PORT=8000
+CRDB_PORT=26257
+CHAOS_DASH_PORT=2333
+JAEGER_DASH_PORT=16686
+MINIO_PORT=9000
+
+# Makefile function. call with $(call,<param1>,<param2>...)
+define forward-port
+		$(if $(filter-out 1,$(FOWARD_TO_LOCAL_NETWORK)),kubectl port-forward $(1) $(2):$(2),kubectl port-forward $(1) --address 0.0.0.0 $(2):$(2))
```

Comment:
I'd prefer if its a separate command to port-forward on all IPs, by default this should just port forward locally, so we're not exposing anything externally unexpectedly

Reply:
so something like `make port-forward-conditionorc-api-lan`?

Reply:
yep, or `port-forward-all-lan`